mmv1/products/monitoring/UptimeCheckConfig.yaml

```diff
@@ -401,3 +409,28 @@ properties:
           Values for all of the labels listed in the associated monitored
           resource descriptor. For example, Compute Engine VM instances use the
           labels "project_id", "instance_id", and "zone".
+  - !ruby/object:Api::Type::NestedObject
+    name: syntheticMonitor
+    immutable: true
```

Comment:
Is ` syntheticMonitor` updatable?

Reply:
at the moment, no. do you think that I should remove this if it _might_ become mutable?

Reply:
Not now. It could be removed after it becomes mutable later.